core/templates/domain/skill/skill-rights-backend-api.service.ts

```diff
@@ -16,92 +16,99 @@
  * @fileoverview Service to change the rights of skills in the backend.
  */
 
-require('domain/utilities/url-interpolation.service.ts');
+export interface ISkillRightCache {
+  [propName: string]: ISkillRightBackend
+}
 
-angular.module('oppia').factory('SkillRightsBackendApiService', [
-  '$http', '$q', 'UrlInterpolationService', 'SKILL_RIGHTS_URL_TEMPLATE',
-  function($http, $q, UrlInterpolationService, SKILL_RIGHTS_URL_TEMPLATE) {
-    // Maps previously loaded skill rights to their IDs.
-    var skillRightsCache = {};
+import { cloneDeep } from 'lodash';
```

Comment:
Move clonedeep below injectable after adding a line break.
```
import { downgradeInjectable } from '@angular/upgrade/static';
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';

import clonedeep

import other stuff
```

Reply:
fixed it